.github/workflows/src/arm-auto-signoff/arm-auto-signoff-status.js

```diff
@@ -5,11 +5,108 @@ import { byDate, invert } from "../../../shared/src/sort.js";
 import { extractInputs } from "../context.js";
 import { LabelAction } from "../label.js";
 
+const LABEL_ARM_SIGNED_OFF = "ARMSignedOff";
```

Comment:
nit: these might be better as an enum containing all labels?  example:

https://github.com/Azure/azure-rest-api-specs/blob/main/.github/workflows/src/label.js